MSAL/src/public/MSALError.h

```diff
@@ -254,75 +229,110 @@ typedef NS_ENUM(NSInteger, MSALErrorCode)
      */
-    MSALErrorNonHttpsRedirect                   = -42602,
-    
-    /*!
-     The requested resource is protected by an Intune Conditional Access policy.
-     The calling app should integrate the Intune SDK and call the remediateComplianceForIdentity:silent: API,
-     please see https://aka.ms/intuneMAMSDK for more information.
-     */
-    MSALErrorServerProtectionPoliciesRequired   = -42603,
+    MSALInternalErrorNonHttpsRedirect                   = -42602,
     
     /*!
      User returned manually to the application without completion authentication inside the broker
      */
-    MSALErrorBrokerResponseNotReceived          = -42700,
+    MSALInternalErrorBrokerResponseNotReceived          = -42700,
     
     /*!
      MSAL cannot read broker resume state. It might be that application removed it, or NSUserDefaults is corrupted.
      */
-    MSALErrorBrokerNoResumeStateFound           = -42701,
+    MSALInternalErrorBrokerNoResumeStateFound           = -42701,
     
     /*!
      MSAL cannot read broker resume state. It is corrupted.
      */
-    MSALErrorBrokerBadResumeStateFound          = -42702,
+    MSALInternalErrorBrokerBadResumeStateFound          = -42702,
     
     /*!
      MSAL cannot read broker resume state. It is saved for a different redirect uri. The app should check its registered schemes.
      */
-    MSALErrorBrokerMismatchedResumeState        = -42703,
+    MSALInternalErrorBrokerMismatchedResumeState        = -42703,
     
     /*!
      Invalid broker response.
      */
-    MSALErrorBrokerResponseHashMissing          = -42704,
+    MSALInternalErrorBrokerResponseHashMissing          = -42704,
     
     /*!
      Corrupted broker response.
      */
-    MSALErrorBrokerCorruptedResponse            = -42705,
+    MSALInternalErrorBrokerCorruptedResponse            = -42705,
     
     /*!
      Decryption of broker response failed.
      */
-    MSALErrorBrokerResponseDecryptionFailed     = -42706,
+    MSALInternalErrorBrokerResponseDecryptionFailed     = -42706,
     
     /*!
      Unexpected broker response hash.
      */
-    MSALErrorBrokerResponseHashMismatch         = -42707,
+    MSALInternalErrorBrokerResponseHashMismatch         = -42707,
     
     /*!
      Failed to create broker key.
      */
-    MSALErrorBrokerKeyFailedToCreate            = -42708,
+    MSALInternalErrorBrokerKeyFailedToCreate            = -42708,
     
     /*!
      Couldn't read broker key. Maybe broker key got wiped from the keychain.
      */
-    MSALErrorBrokerKeyNotFound                  = -42709,
-    
-    // Workplace join is required to proceed
-    MSALErrorWorkplaceJoinRequired              = -42710,
+    MSALInternalErrorBrokerKeyNotFound                  = -42709,
     
     /*!
      Broker returned unreadable result.
      */
-    MSALErrorBrokerUnknown                      = -42711
+    MSALInternalErrorBrokerUnknown                      = -42711
+};
+
+typedef NS_ENUM(NSInteger, MSALError)
```

Comment:
nit: should this enum be the first one in the file for discoverability?

Reply:
done